tsconfig.json

```diff
@@ -1,6 +1,11 @@
 {
   "compilerOptions": {
-    "strict": true
+    "strict": true,
+    "module": "commonjs",
+    "target": "es2019",
+    "lib": ["es2020"],
+    "noImplicitReturns": true,
+    "noUnusedLocals": true
```

Comment:
This is already checked by ESLint.

Reply:
The last 2 I guess. The first 2 refer to the generated code

Reply:
The last one for sure, the third one probably.